src/assets/js/components/session.tsx

```diff
@@ -83,12 +87,25 @@ export default class SessionComponent extends React.Component<Props, State> {
     // make true to output time taken to get render contents
     this.profileRender = true;
 
+    this.getProfiler = (profileRender) => {
+      if (!profileRender) {
+        return () => () => null;
+      }
+
+      let t0 = Date.now();
```

Comment:
this line should go away - we want different t0 for every run of profiling to avoid races